### PR TITLE
Updates several pacakges and adapts code

### PR DESCRIPTION
**Pull request by KaiVolland**
This updates several packages and adapts the corresponding code.

closes #117 
closes #116 
closes #115 
closes #110 
closes #109 
closes #108 
closes #107 
closes #106 
closes #105 
closes #96

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -69,6 +68,12 @@ interface AppState {
 
 class App extends React.Component<AppProps, AppState> {
 
+  _sldStyleParser = new SldStyleParser();
+
+  _geoJsonParser = new GeoJsonParser();
+
+  _wfsParser = new WfsParser();
```
Comment (dnlkoch) [inline src/App.tsx]: Absolutely minor, but one could use the `private` keyword to highlight nature of these members.

File: src/App.tsx [hunk 2]
```diff
@@ -41,6 +32,14 @@ import {
   StyleLoader
 } from 'geostyler';
 
+import logo from './assets/logo.svg';
+import 'antd/dist/antd.css';
```
Comment (dnlkoch) [inline src/App.tsx]: Purely a matter of taste, but I would import the framework specific CSS classes in the [index.css](https://github.com/terrestris/geostyler-demo/blob/master/src/index.css) (-> `@import url("antd/dist/antd.css");`). You decide.

Comment (KaiVolland) [reply src/App.tsx]: Didn't manage to do the import in css with an relativ url. `@import url("../node_modules/antd/dist/antd.css");` worked for the dev setup but i didn't like to include the `node_modules` in the path.

**Pull request merged**